packages/fxa-react/components/LoadingSpinner/index.tsx

```diff
@@ -4,7 +4,7 @@ import { ReactComponent as Spinner } from './spinner.svg';
 export const LoadingSpinner = ({ className }: { className?: string }) => (
   <div {...{ className }} data-testid="loading-spinner">
     <Spinner
-      className="spinner-image w-10 h-10 animate-rotate animate-timing-linear animate-duration-800 animate-repeat-infinite"
+      className="w-10 h-10 animate-spin"
```

Comment:
We had all of these classes in a separate `animations.css` file and they were only used on the loading spinner. I added the `spin` animation to the TW config file and removed that file.